spec/controllers/miq_ae_tools_controller_spec.rb

```diff
@@ -356,8 +359,22 @@
           end
         end
 
+        context "when the repository is using self signed certificates" do
+          let (:verify_ssl) { OpenSSL::SSL::VERIFY_NONE }
+          let (:git_verify_ssl) { "false" }
+          before do
+            allow(GitRepository).to receive(:create).with(:url => git_url, :verify_ssl => verify_ssl).and_return(git_repo)
+            allow(GitRepository).to receive(:exists?).with(:url => git_url).and_return(false)
+          end
```

Comment:
Another line break here as well. It helps to break up the different sections of the context and keeps it consistent with the rest of the file.